content/docs/pipelines/reusable-components.md

```diff
@@ -0,0 +1,71 @@
++++
+title = "Index of Reusable Components"
+description = "A list of Kubeflow Pipelines components that you can use in your pipelines"
+weight = 2
++++
+
+A Kubeflow Pipelines *component* is a self-contained set of code that performs 
+one step in the pipeline, such as data preprocessing, data transformation, model
+training, and so on. Each component is packaged as a Docker image.
+You can add existing components to your pipeline. These may be components that
+you create yourself, or that someone else has created and made available.
+
+The Kubeflow Pipelines repository includes a variety of 
+[reusable components](https://github.com/kubeflow/pipelines/tree/master/components)
+that you can add to your pipeline. This page highlights the components that
+include usage documentation in the form of README files.
+
```

Comment:
I've added an introductory sentence to each section explaining the GCP connection. I don't want to put GCP right at the top of the page, because I hope we'll be able to add non-GCP components sometime soon.